src/routes/v1/article.route.js

```diff
@@ -0,0 +1,30 @@
+import { Router } from 'express';
+import ArticleController from '../../controllers/articles.controller';
+import authMiddleware from '../../middleware/auth.middleware';
+import validationMiddleware from '../../middleware/validation.middleware';
+
+const router = Router();
+const validateRequest = validationMiddleware();
+
+router.post(
+  '/articles',
+  authMiddleware,
+  validateRequest,
+  ArticleController.createArticle
+);
+router.get('/articles', ArticleController.fetchAllArticles);
+router.get('/articles/:articleId', ArticleController.fetchSpecificArticle);
+router.put(
+  '/articles/:articleId',
+  authMiddleware,
+  validateRequest,
+  ArticleController.updateOneArticle
+);
+
+router.delete(
```

Comment:
I think you should validate the `articleId` here, this might lead to a bug. What happens when the user enters a string as the articleId ?

Reply:
Thank you for this review.
However, I'd like you to be specific on where you're pointing to as to enable me to make effective changes.
There is validation for the `articleId` and may be there is an oversight on some routes where I forgot to introduce the validations.
Guess that is what you are pointing to?

Reply:
Thank you @tolumide-ng and @codexempire for this feedback.
Their validation for the `articleId` implemented on the `validation schema`